.circleci/config.yml

```diff
@@ -27,7 +33,7 @@ jobs:
       - run: wget -v --retry-connrefused -T 5 -O /dev/null 127.0.0.1:8081
       - run: |
           # async send to zipkin, so wait a bit before reading back
```

Comment:
ps this is a bit of a lie.. in the in-memory storage, it is synchronous (no threads are used)